src/app/js/public/MenuItem.js

```diff
@@ -0,0 +1,297 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import { Link, NavLink } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import * as icons from '@fortawesome/free-solid-svg-icons';
+import get from 'lodash.get';
+import { StyleSheet, css } from 'aphrodite/no-important';
+import classnames from 'classnames';
+
+import theme from '../theme';
+import { polyglot as polyglotPropTypes } from '../propTypes';
+
+const styles = StyleSheet.create({
+    link: {
+        textDecoration: 'none',
+        ':hover': {
+            textDecoration: 'none',
+        },
+        ':focus': {
+            textDecoration: 'none',
+            color: theme.orange.primary,
+        },
+        ':visited': {
+            textDecoration: 'none',
+        },
+        ':active': {
+            color: theme.orange.primary,
+        },
+    },
+    active: {
+        color: theme.orange.primary,
+        ':hover': {
+            color: theme.orange.primary,
+        },
+    },
+    drawerActive: {
+        color: `${theme.purple.primary} !important`,
+        ':hover': {
+            color: `${theme.purple.primary} !important`,
+        },
+    },
+    menuItem: {
+        width: '100%',
+        height: 75,
+        padding: 10,
+        display: 'flex',
+        flexDirection: 'column',
+        textAlign: 'center',
+        color: theme.green.primary,
+        cursor: 'pointer',
+        justifyContent: 'center',
+        userSelect: 'none',
+        textTransform: 'capitalize',
+        ':hover': {
+            color: theme.purple.primary,
+        },
+        ':active': {
+            color: theme.orange.primary,
+        },
+    },
+    menuItemIcon: {
+        margin: '0 auto',
+    },
+});
+
+const getIcon = icon => {
+    const faIcon = get(icons, icon);
+    if (faIcon) {
+        return (
+            <FontAwesomeIcon
+                icon={faIcon}
+                className={css(styles.menuItemIcon)}
+            />
+        );
+    }
+    return (
+        <img
+            src={icon}
+            className={css(styles.menuItemIcon)}
+            width={50}
+            height={50}
+        />
+    );
+};
+
+const MenuItem = ({
+    config,
+    polyglot,
+    closeAll,
+    hasGraph,
+    graphDrawer,
+    searchDrawer,
+    handleGraphItemClick,
+    toggleSearch,
+    canBeSearched,
+    role,
+    logout,
+}) => {
+    const label = config.label[polyglot.currentLocale];
+    const icon = getIcon(config.icon);
+
+    switch (config.role) {
```

Comment:
You should use `toLowerCase` here, just to be sure

Reply:
nope, i think it's better to be strict for config